kafka-ui-react-app/src/components/common/Dropdown/Dropdown.tsx

```diff
@@ -20,7 +20,7 @@ const Dropdown: React.FC<DropdownProps> = ({ label, right, up, children }) => {
       cx('dropdown', {
         'is-active': active,
         'is-right': right,
-        'is-up': up,
+        'is-up': true,
```

Comment:
here you should pass up true in this case instead of hardcoding it into the component which is used several places , you just go to this table page and pass the up prop to the dropdown